cookbook/deployment/customizing_resources.py

```diff
@@ -48,21 +50,59 @@ def count_unique_numbers(x: typing.List[int]) -> int:
 
 
 # %%
-# Let's create a dummy task that determines the square of a number.
+# Define a task that computes the square of a number.
 @task
 def square(x: int) -> int:
     return x * x
 
 
 # %%
-# The tasks decorated with memory and storage hints can be used like regular tasks in a workflow.
+# The tasks decorated with memory and storage hints are used like regular tasks in a workflow.
 @workflow
 def my_workflow(x: typing.List[int]) -> int:
     return square(x=count_unique_numbers(x=x))
 
 
 # %%
-# The workflow and task can be executed locally.
+# You can execute the workflow locally.
 if __name__ == "__main__":
     print(count_unique_numbers(x=[1, 1, 2]))
     print(my_workflow(x=[1, 1, 2]))
+
+# %%
+# Using ``with_overrides`` Method
+#
+# The ``with_overrides`` method is used to override the resources in the tasks based on the inputs.
+# Let's understand it with an example
+#
+
+# %%
+# Import the dependencies.
+from flytekit import task, workflow, Resources
+
+# %%
+# Define a task that returns an integer and a string.
+@task
+def t1(a:int, b:str) -> (int, str):
+    return (a,b)
+
+# %%
+# Define a task that returns sum of two integers.
+@task
+def t2(c:int, d:int) -> int:
+    return c+d
+
+# %%
+# Define a workflow. 
```

Comment:
It'd be nice if you talk a little about with_overrides; one line should suffice.

Reply:
Isn't "The ``with_overrides`` method is helpful to override the resources allocated to a task dynamically." relevant? It is present at the beginning.